samples/Samples.h

```diff
@@ -108,12 +112,16 @@ typedef struct {
     UINT32 iceUriCount;
     SignalingClientCallbacks signalingClientCallbacks;
     SignalingClientInfo clientInfo;
+    MediaStorageConfig mediaStorageConfig;
+    UINT32 mediaStorageClipLength;
```

Comment:
mediaStorageConfig stores the information of media storage, and mediaStorageClipLength stores the length of clips. It will be used in kvsWebRTCClientStorageSession.c, but mediaStorageClipLength will be the parameter to decide the sample as  persistent video streaming or periodic video streaming.